components/collapsible-panel/README.md

```diff
@@ -0,0 +1,87 @@
+# Collapsible Panel
+
+A collapsible panel is a container that can be expanded and collapsed to show/hide additional content and form options. Content within the panel is flexible and customizable -- form controls, buttons, text, and more can be put in the expanded version of this panel for users to interact with or view.
+
+## Best Practices
+
+<!-- docs: start best practices -->
+<!-- docs: start dos -->
+* Use for optional settings or additional information that you want to initially hide
+* Use when the user would benefit from progressively disclosed information
+* Always use this in the right hand panel FACE Activity Creation experience (FACE, First Use Activity Create & Edit)
+<!-- docs: end dos -->
+
+<!-- docs: start donts -->
+* Don't nest collapsible panels within each other
+* Avoid using a collapsible panel for required options in a form
+* Avoid long lists of collapsible panels; consider using the nested list in this case (exceptions may apply)
+* Don't have interactions or elements in the closed state that are different or disappear in the open state (exception: the summary)
+	* For example, don't have a button that's only available in the collapsed state of the panel
+<!-- docs: end donts -->
+<!-- docs: end best practices -->
+
+## Composition
+
+
+## Collapsible Panel [d2l-collapsible-panel]
+
+The `d2l-collapsible-panel` element is a container that provides specific layout slots such as `header`, `summary`, `actions`, and a default slot for the expanded content.
+
+<!-- docs: demo live name:d2l-collapsible-panel size:large -->
+```html
+<script type="module">
+	import '@brightspace-ui/core/components/button/button-subtle.js';
+  import '@brightspace-ui/core/components/collapsible-panel/collapsible-panel.js';
+  import '@brightspace-ui/core/components/collapsible-panel/collapsible-panel-summary-item.js';
```

Comment:
Nit: indentation

Reply:
Yup 😄 demo file is still somewhat volatile